src/helpers/dates.ts

```diff
@@ -714,3 +714,22 @@ export function isDateStrictlyAfter(date: number, dateAfter: number) {
 export function isDateAfter(date: number, dateAfter: number) {
   return Math.trunc(date) >= Math.trunc(dateAfter);
 }
+
+export function weekNumber(date: DateTime, startDayOfWeek: number): number {
```

Comment:
Is there a reason to extract it ? It seems to only be used in `module_date`...
If there is a good reason, take the opportunity to write some unit test on this function :D